sigstore-java/src/test/java/dev/sigstore/encryption/KeysTest.java

```diff
@@ -184,6 +185,18 @@ void parseTufPublicKey_rsa()
     }
   }
 
+  @Test
+  void parsePublicKeyBad()
```

Comment:
So yeah these kind of negative tests in junit5 should follow the pattern
```
  // add a comment to the link to oss-fuzz test case?
  @Test
  void parsePublicKey_failOnNullSection() {
    byte[] key = "-----BEGIN A-----\nBBBBB-----END A".getBytes(StandardCharsets.UTF_8);
    Assertions.assertThrows(InvalidKeySpecException.class, () -> Keys.parsePublicKey(key));
  }
```

Reply:
you can save the result from Assertions.assertThrows and compare the message in the Exception to what's expected too, but I don't think that's necessary here in this case.

Reply:
Thanks for the suggestion. I am thinking the opposite, we want the unit testing to assert that the NPE is not thrown with the fix, right? Thus Assertions.assertThrows may not be the one we want to use, instead the fail() should exist in the catch block if the exception caught is a NPE. In other words, I am assuming that the unit testing should fail for throwing NPE before the fix and pass after the fix.

Reply:
Thanks for the suggestion. Changed accordingly.